flow/util/genMetrics.py

```diff
@@ -102,7 +84,7 @@ def extractTagFromFile(
 
     # Open file
     try:
-        searchFilePath = os.path.join(args.flowPath, file)
+        searchFilePath = file
```

Comment:
It looks like we don't need this variable anymore with the change.

Reply:
true, I just wanted to keep the PR small

Reply:
leave this PR as is and refine more later? 

I dont want to mix concerns in a PR

Reply:
Sure, I can pick up the rest of the change.